cla_backend/libs/eligibility_calculator/models.py

```diff
@@ -1,34 +1,44 @@
+import types
+
 from . import exceptions
 
 
 class ModelMixin(object):
     PROPERTY_META = None
 
+    @staticmethod
+    def is_a_subclass_of_model_mixin(property_meta_value):
```

Comment:
Feels a bit prominant to have this method first in the class - maybe move it to be last?